docs/modules/content/developer-guides/search-helper/index.md

```diff
@@ -0,0 +1,100 @@
+# Search helper package
+De search helper package biedt de mogelijkheid om bij het registeren van een nieuw module 
+een versimpelde manier om nieuwe mappers toe te voegen. 
+Zowel content als search mappers kunnen toegevoegd worden.
+
+
+## Voorbereiding
+Om deze gids te kunnen volgen heb je de volgende zaken nodig
+- Je hebt De Redactie reeds [lokaal draaien](/modules/content/setup/redactie/setup).
+- Je hebt het WCM reeds [lokaal draaien](/modules/content/setup/wcm/index.md)
+
+## Installatie
+Om de search helper package te installeren voer je volgend commando uit:
+```bash
+npm install @wcm/images-bsl-helper
+```
+## De instantie aanmaken
+In mijn voorbeeld maak ik gebruik van nestjs. Bij de opstart van de module en wanneer de tenantConfig klaar is met registreren van de modules, registreer ik de search helper package met de nieuw helpers. 
```

Comment:
Best niet vanuit de ik vorm praten:
In dit voorbeeld wordt er gebruik gemaakt van NestJS. Bij de opstart van de module en wanneer de tenantConfig klaar is met registreren van de modules, wordt de search helper package met de nieuw helpers geregistreerd.